### PR TITLE
modifies prepare.js

### PR DESCRIPTION
**Pull request by juancho11gm**
Modifies prepare.js to paste src contents to public folder

### CODE REVIEW

File: prepare.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const rimraf = require('rimraf');
+const ncp = require('ncp').ncp;
+var fs = require('fs');
+var public_dir = './public';
```
Comment (fabianmarinog) [inline prepare.js]: This should be a const: `const public_dir = './public';`

File: prepare.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    ncp('./src/assets', './public/assets', function (ncp_error) {
+        if (ncp_error) { throw ncp_error; }
+        // done
+    });
+
+    ncp('./src/images', './public/images', function (ncp_error) {
+        if (ncp_error) { throw ncp_error; }
+        // done
+    });
```
Comment (fabianmarinog) [inline prepare.js]: create a function that can be reused so you don't need to duplicate code

Comment (fabianmarinog) [inline prepare.js]:
```suggestion
const file_list = [
{fileSource: './src/index.html', fileDest: './public/index.html'},
    {fileSource: './src/form.html', fileDest:  './public/form.html'},
    {fileSource:'./src/404.html', fileDest: './public/404.html'},
    {fileSource:'./src/assets', fileDest: './public/assets'},
    {fileSource:'./src/images', fileDest: './public/images'}
];
function copy_files(sourcePath, destPath){
    ncp(sourcePath, destPath, function (ncp_error) {
        if (ncp_error) { throw ncp_error; }
    });
}
file_list.forEach((fileElement, index, array) => {
   const { fileSource, fileDest } = fileElement;
    copy_files(fileSource, fileDest);
});
```

File: .travis.yml [hunk 3]
```diff
@@ -5,6 +5,8 @@ script:
  - echo "Deploy!!"
 install:
   - npm install -g firebase-tools
+  - npm install rimraf
+  - npm install ncp
```
Comment (fabianmarinog) [inline .travis.yml]:
```suggestion
  - npm install
```

Comment (fabianmarinog) [reply .travis.yml]: This command will install all the depedencies from your package.json so you don't have to run the same command for each one.

File: package.json [hunk 4]
```diff
@@ -3,8 +3,13 @@
...
   "description": "",
   "scripts": {
-    "prepare-public": "cp src/elements.html public/elements.html"
+    "prepare-public": "node prepare.js"
   },
   "author": "push-dev",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "mkdirp": "^1.0.3",
```
Comment (fabianmarinog) [inline package.json]: is `mkdirp` being used? if not, remove this reference.

File: prepare.js (hunk 1 above)
Comment (juancho11gm) [reply prepare.js]: Done

File: prepare.js (hunk 2 above)
Comment (juancho11gm) [reply prepare.js]: Sure!

File: .travis.yml (hunk 3 above)
Comment (juancho11gm) [reply .travis.yml]: ok!

File: package.json (hunk 4 above)
Comment (juancho11gm) [reply package.json]: ok!

**Pull request merged**